src/genetic_testing/routers/ncbi.py

```diff
@@ -28,6 +29,8 @@
     "Status",
 ]
 RETMAX = 10000
+RETTYPE = "fasta"
+RETMODE = "text"
```

Comment:
Please add a small comment on what these constants are related to.

Reply:
Yes, my bad.